app/views/comments/index.json.jbuilder

```diff
@@ -0,0 +1,5 @@
+json.array!(@resource_comments) do |comment|
+  json.partial! 'comments/default', data: comment
+  json.extract! comment, :comment_on_id, :comment_on_type
+  json.partial! 'users/user', user: comment.user
```

Comment:
You're correct. I obviously did not pay attention to details there. Although, `jbuilder` would not render it twice. Corrected.